Dockerfile

```diff
@@ -6,15 +6,16 @@ RUN sudo bash -c "echo \"deb http://ftp.us.debian.org/debian testing main contri
   && sudo apt-get update \
   # bzip2 and libgconf-2-4 are necessary for extracting firefox and running chrome, respectively
   && sudo apt-get install bzip2 libgconf-2-4 node-less cmake build-essential clang-format-6.0 \
-                  uglifyjs chromium ccache libncurses6 gfortran f2c \
+                  uglifyjs chromium ccache libncurses6 gfortran f2c swig \
   && sudo apt-get install -t testing g++-8 \
   && sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-6 60 --slave /usr/bin/g++ g++ /usr/bin/g++-6 \
   && sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 80 --slave /usr/bin/g++ g++ /usr/bin/g++-8 \
   && sudo update-alternatives --set gcc /usr/bin/gcc-8 \
   && sudo apt-get clean \
   && sudo apt-get autoremove \
   && sudo ln -s /usr/bin/clang-format-6.0 /usr/bin/clang-format \
-  && sudo bash -c "echo 'application/wasm wasm' >> /etc/mime.types"
+  && sudo bash -c "echo 'application/wasm wasm' >> /etc/mime.types" \
+  && sudo sed -ri "1 s|^#\!.*|#\!$(which node)|" "$(realpath "$(which uglifyjs)")"
```

Comment:
I think there was indeed some issue with uglifyjs , could you please add a comment on what this does?

Reply:
Argh, forgot to commit the comment:
With the current version from the debian repos, uglifyjs switched from using a hardcoded nodejs path to finding nodejs from `$PATH`. This means that uglifyjs is suddenly invoked with the nodjs in emscripten, and that doesn't have the necessary dependencies.

It's a terrible hack, but I don't think forcing apk to install an old version is much cleaner. I'd be thankful for advice. Maybe I should write a uglifyjs wrapper that cleans emscripten (and itself) from path?

Reply:
> Maybe I should write a uglifyjs wrapper that cleans emscripten (and itself) from path?

Hmm, I'm not too familiar with node. Isn't there a way to force an executable to run with a particular version of  node? A wrapper that cleans up the PATH is also a solution.